### DISCUSSION TITLE
Mark content which is moved and not deleted with yellow

### DISCUSSION THREAD

**Discussion by cieniawska** [community/community#15190] [Pull Requests]
Currently, when reviewing pull requests there is a distinction, when:
- content has been deleted - it's red,
- content has been added - it's green.

However, when the content is moved - f.ex in one file a line is moved to the top - it is displayed both as red (on the left) and green (on the right) which is a bit confusing since it was only moved. 

My suggestion is that it should be differentiated with another colour, f.ex yellow.

**Comment by SPConrad** [comment]
Imagine my surprise when I came here to suggest this feature and found someone suggested the same thing just three days ago! I was thinking gray, but yellow would be just fine, too. Heck, any color at all. 

This would be immensely helpful with PRs that currently appear large but are in reality moderately or even significantly smaller. Moved code should still appear within the PR, of course, but it would be greatly beneficial to be able to focus on the "true" additions and removals.

**Reply by brendanwalters** [reply]
See also earlier request #9632
